client/packages/programs/src/JsonForms/common/components/Autocomplete.tsx

```diff
@@ -52,7 +52,15 @@ const UIComponent = (props: ControlProps) => {
     () => getDisplayOptions(schemaOptions),
     [schemaOptions]
   );
+
+  const clearable = !props.config?.required;
+
   useEffect(() => {
+    if (!data && clearable) {
```

Comment:
does data update when handleChange is called (from parent) ?

Reply:
Yep